pkg/target/target_builder.go

```diff
@@ -204,6 +204,14 @@ func (b *targetBuilderImpl) completeTargetForShoot(ctx context.Context, t *targe
 		return fmt.Errorf("failed to fetch shoot: %w", err)
 	}
 
+	if handler := ac.AccessRestrictionHandlerFromContext(ctx); handler != nil {
+		if garden, err := b.config.Garden(t.GardenName()); err == nil {
```

Comment:
same here, why don't we return on error?

Reply:
✔️